benchmarks/clue/run_clue.sh

```diff
@@ -30,12 +30,13 @@ if [ "$mode" = "train" ]; then
     --label_name=label \
     --label_enumerate_values=0,1 \
     --checkpoint_dir=./tmp/benchmarks/clue/$task \
+    --optimizer_type=AdamW \
```

Comment:
change it to:     --optimizer=AdamW \

Reply:
split it to two shell scripts:
run_clue_adamw.sh
run_clue_bertadam.sh

Reply:
done